backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -0,0 +1,104 @@
+
+            if (filter.NELatitude.HasValue && filter.NELongitude.HasValue && filter.SWLatitude.HasValue && filter.SWLongitude.HasValue)
+                query = query.Where(b =>
+                    b.Latitude != 0 &&
+                    b.Longitude != 0 &&
+                    b.Latitude <= filter.NELatitude &&
+                    b.Latitude >= filter.SWLatitude &&
+                    b.Longitude <= filter.NELongitude &&
+                    b.Longitude >= filter.SWLongitude);
+
+            if (filter.Agencies?.Any() == true)
+            {
+                // Get list of sub-agencies for any agency selected in the filter.
+                var agencies = filter.Agencies.Concat(context.Agencies.AsNoTracking().Where(a => filter.Agencies.Contains(a.Id)).SelectMany(a => a.Children.Select(ac => ac.Id)).ToArray()).Distinct();
+                query = query.Where(p => agencies.Contains(p.AgencyId));
+            }
+            if (filter.ClassificationId.HasValue)
+                query = query.Where(p => p.ClassificationId == filter.ClassificationId);
+            if (filter.StatusId.HasValue)
+                query = query.Where(p => p.StatusId == filter.StatusId);
+            if (!String.IsNullOrWhiteSpace(filter.ProjectNumber))
+                query = query.Where(p => EF.Functions.Like(p.ProjectNumber, $"{filter.ProjectNumber}%"));
+            if (!String.IsNullOrWhiteSpace(filter.Description))
+                query = query.Where(p => EF.Functions.Like(p.Description, $"%{filter.Description}%"));
+
+            if (!String.IsNullOrWhiteSpace(filter.Municipality))
+                query = query.Where(p => EF.Functions.Like(p.Municipality, $"%{filter.Municipality}%"));
+            if (!String.IsNullOrWhiteSpace(filter.Zoning))
+                query = query.Where(p => EF.Functions.Like(p.Zoning, $"%{filter.Zoning}%"));
+            if (!String.IsNullOrWhiteSpace(filter.ZoningPotential))
+                query = query.Where(p => EF.Functions.Like(p.ZoningPotential, $"%{filter.ZoningPotential}%"));
+
+            if (filter.ConstructionTypeId.HasValue)
+                query = query.Where(p => p.BuildingConstructionTypeId == filter.ConstructionTypeId);
+            if (filter.PredominateUseId.HasValue)
+                query = query.Where(p => p.BuildingPredominateUseId == filter.PredominateUseId);
+            if (filter.FloorCount.HasValue)
+                query = query.Where(p => p.BuildingFloorCount == filter.FloorCount);
+            if (!String.IsNullOrWhiteSpace(filter.Tenancy))
+                query = query.Where(p => EF.Functions.Like(p.BuildingTenancy, $"%{filter.Tenancy}%"));
+
+            if (!String.IsNullOrWhiteSpace(filter.Address)) // TODO: Parse the address information by City, Postal, etc.
+                query = query.Where(p => EF.Functions.Like(p.Address, $"%{filter.Address}%") || EF.Functions.Like(p.City, $"%{filter.Address}%"));
```

Comment:
I'm not an expert on the backend so I might be wrong but shouldn't the above be:

`query = query.Where(p => EF.Functions.Like(p.Address.[Address1], $"%{filter.Address}%")`

_[Address1] added by me_

Reply:
yeah, this is what I was getting to in my comment as well, I thought the address and city were discrete fields.

Reply:
I think you're right about the `Address.Address` comment.

Reply:
The view flattens the values into Address.

Reply:
I see, given that both Devin and I were kind of confused by the change in behavior would you consider dropping a comment there for future reference? thanks

Reply:
I can add the comment in the next PR.  This specific query is against the new View (of which the property Address is flattened slightly).